### PR TITLE
Heatmap pdf export.

### PR DESCRIPTION
**Pull request by desilvaNSP**

### CODE REVIEW

File: server/webapps/dashboard/components/dashboard-apreport.html [hunk 1]
```diff
@@ -358,8 +404,39 @@
                 opacity: $("#opacity-slider").slider("value")
             });
             heatmap.setMap(map);
+            setGradient();
+            setLegendGradient();
         }
+        function setGradient() {
+            heatmapgradient = [
+                'rgba(14, 249, 11,0)',
```
Comment (desilvaNSP) [inline server/webapps/dashboard/components/dashboard-apreport.html]: Change the gradient color on heatmap according to the ISL request. green to red

Comment (apremalal) [inline server/webapps/dashboard/components/dashboard-apreport.html]: But they wanted yellow as well?

**Comment by apremalal** [comment]
Again thanks for the hard work and this is great work, we need to write few blog post on some of these stuff when we get some time. 

There's a small issue on the PDF, see the attached for details. The titles are overlapping.

**Comment by apremalal** [comment]
issue

**Pull request closed without merging**